.github/CODEOWNERS

```diff
@@ -908,6 +908,12 @@
 # ServiceLabel: %Playwright
 # ServiceOwners:                                                   @shreyaanand @mjmadhu
 
+# PRLabel: %Communication - programmableconnectivity
```

Comment:
Please move up to preserve alphabetical order.

Reply:
So the order is currently:

```
/sdk/appconfiguration/
/sdk/attestation/Azure.Security.Attestation
/sdk/azurestack*/
/sdk/batch/
/sdk/cognitivelanguage/
/sdk/contentsafety/
/sdk/cognitiveservices/Vision.CustomVision*/
/sdk/cognitiveservices/Vision.ComputerVision/
/sdk/cognitiveservices/Vision.Face/
/sdk/documentintelligence/
/sdk/formrecognizer/
/sdk/metricsadvisor/
/sdk/anomalydetector/
/sdk/healthinsights/
/sdk/cognitiveservices/Knowledge.QnAMaker/
/sdk/textanalytics/
/sdk/translation/
/sdk/cognitiveservices/Language.LUIS*/
/sdk/cognitiveservices/Search.Bing*/
/sdk/communication/
/sdk/communication/Azure.Communication.CallingServer/
/sdk/communication/Azure.Communication.Chat/
/sdk/communication/Azure.Communication.Common/
/sdk/communication/Azure.Communication.Identity/
/sdk/communication/Azure.Communication.NetworkTraversal/
/sdk/communication/Azure.Communication.PhoneNumbers/
/sdk/communication/Azure.Communication.ShortCodes/
/sdk/communication/Azure.Communication.Sms/
/sdk/communication/Azure.Communication.Rooms/
/sdk/compute/
/sdk/confidentialledger/
/sdk/containerregistry/
/sdk/cosmosdb/
/sdk/devcenter/
/sdk/deviceupdate/
/sdk/digitaltwins/
/sdk/easm/Azure.Analytics.Defender.Easm/
/sdk/eventgrid/
/sdk/eventgrid/Microsoft.Azure.WebJobs.Extensions.EventGrid/
/sdk/eventhub/
/sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/
/sdk/eventhub/Microsoft.Azure.EventHubs/
/sdk/eventhub/Microsoft.Azure.EventHubs.Processor/
/sdk/eventhub/Microsoft.Azure.EventHubs.ServiceFabricProcessor/
/sdk/hdinsight/
/sdk/iot*/
/sdk/modelsrepository/
/sdk/keyvault/
/sdk/loadtestservice/
/sdk/maps/
/sdk/mediaservices/
/sdk/monitor/ci.yml
/sdk/monitor/Azure.Monitor.Ingestion/
/sdk/monitor/Azure.Monitor.Query/
/sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/
/sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/
/sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/
/sdk/mixedreality/
/sdk/objectanchors/
/sdk/network*/
/sdk/dns*/
/sdk/nginx/
/sdk/openai/
/sdk/vision/Azure.AI.Vision.ImageAnalysis/
/sdk/quantum/Azure.Quantum.Jobs/
/sdk/search/
/sdk/servicebus/
/sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/
/sdk/schemaregistry/
/sdk/signalr/
/sdk/storage*/
/sdk/storage/Azure.Storage.*/
/sdk/storage/Microsoft.Azure.WebJobs.*/
/sdk/synapse/
/sdk/tables/
/sdk/timeseriesinsights/
/sdk/videoanalyzer/
/sdk/webpubsub/
/sdk/webpubsub/Microsoft.Azure.WebJobs.Extensions.WebPubSub/
/sdk/personalizer/
/sdk/playwrighttesting/
/sdk/programmableconnectivity/
```

So there's already quite a lot of ones out of order, but from what I can tell, the ones earlier should be on top. 

`programmableconnectivity` comes after `personalizer` and `playwrighttesting`, so I'm going to leave as is.

Reply:
They're organized by the name of the label, not the path.   This would belong up with the other Communication tags.